samples/client/petstore/typescript-node/default/api/petApi.ts

```diff
@@ -83,17 +83,17 @@ export class PetApi {
     /**
      * 
      * @summary Add a new pet to the store
-     * @param body Pet object that needs to be added to the store
+     * @param pet Pet object that needs to be added to the store
      */
-    public async addPet (body: Pet, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.ClientResponse; body?: any;  }> {
+    public async addPet (pet: Pet, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.ClientResponse; body?: any;  }> {
```

Comment:
please run the [bin/typescript-node-petstore-all.sh](https://github.com/OpenAPITools/openapi-generator/blob/master/bin/typescript-node-petstore-all.sh) script only, not the `openapi3` version